docs/engineering/technical-planning.md

```diff
@@ -0,0 +1,27 @@
+# Technical Planning Docs
+## Request for Comment
+In the course of planning development, there are often scenarios where a technology or implementation decision fork is presented and an engineer wants to solicit ideas or feedback. 
+
+Writing a full-blown Architectural Decision Record may not appropriate as that document requires significant overhead in proposing answers that still may be largely unknown.
```

Comment:
Speaking from experience, I want to write a ADR only when a decision has been made. Otherwise it's a Decision Proposal, not a Decision Record.

Reply:
I agree that a design proposal should be separate. I have commonly found multiple decision points will need to be made with a new design.